x/evm/msg.go

```diff
@@ -148,7 +142,7 @@ func (msg *MsgEthereumTx) FromEthereumTx(tx *gethcore.Transaction) error {
 func (msg MsgEthereumTx) Route() string { return RouterKey }
 
 // Type returns the type value of an MsgEthereumTx.
-func (msg MsgEthereumTx) Type() string { return TypeMsgEthereumTx }
+//func (msg MsgEthereumTx) Type() string { return TypeMsgEthereumTx }
```

Comment:
```suggestion
func (msg MsgEthereumTx) Type() string { return TypeMsgEthereumTx }
```

Looks like this was commented by mistake. If not please, disregard my suggestion and we can remove these lines.

Reply:
We can add a compile-time assertion here to make sure we don't break the interface. The code would be along the lines of:

```go
import (
  "github.com/cosmos/cosmos-sdk/x/auth/migrations/legacytx"
)

// ensure Msg interface compliance at compile time
var (
  _ legacytx.LegacyMsg = &MsgEthereumTx
)
```

This way the compiler will get upset if we remove this method.